WordPress/Classes/Utility/Media/MediaFileManager.swift

```diff
@@ -196,6 +196,18 @@ class MediaFileManager: NSObject {
         cacheManager.calculateSizeOfDirectory(onCompletion: onCompletion)
     }
 
+    /// Helper method for calculating the size of the Media directories.
+    ///
+    class func calculateSizeOfMediaDirectories(onCompletion: @escaping (Int64?) -> Void) {
```

Comment:
Can we remove the `calculateSizeOfMediaCacheDirectory` method now? I don't think we're using it anywhere.